### ISSUE TITLE
support `MODULE.bazel` for identifying workspace root

### ISSUE THREAD

**Issue by klandergren** [bazelbuild/bazel-watcher#646]
**Describe the bug**
`ibazel` does not appear to support finding the workspace root of a project with _just_ a `MODULE.bazel` file (e.g. no `WORKSPACE` or `WORKSPACE.bazel`). As far as I can tell this is a valid repo structure for bazel 7. 

**Reproduction instructions**
[klandergren/ibazel-no-workspace-file](https://github.com/klandergren/ibazel-no-workspace-file) shows a minimal go project that works using bazel 7 but does not work with `ibazel` Version 0.24.0.

1. `git clone https://github.com/klandergren/ibazel-no-workspace-file`
2. `ibazel run //hello:hello`
3. See error

```console
iBazel [6:27PM]: Querying for files to watch...
iBazel [6:27PM]: Error finding workspace: ibazel was not invoked from within a workspace

iBazel [6:27PM]: Error querying for source files: ibazel was not invoked from within a workspace

iBazel [6:27PM]: Error finding workspace: ibazel was not invoked from within a workspace

iBazel [6:27PM]: Error querying for source files: ibazel was not invoked from within a workspace

iBazel [6:27PM]: Running //hello:hello
Computing main repo mapping:
Loading:
Loading: 0 packages loaded
Analyzing: target //hello:hello (0 packages loaded, 0 targets configured)
INFO: Analyzed target //hello:hello (0 packages loaded, 671 targets configured).
 checking cached actions
INFO: Found 1 target...
 checking cached actions
 checking cached actions

INFO: Elapsed time: 0.373s, Critical Path: 0.00s
 checking cached actions
INFO: 0 processes.
 checking cached actions
INFO: Build completed successfully, 0 total actions
INFO: Analyzed target //hello:hello (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
Target //hello:hello up-to-date:
  bazel-bin/hello/hello_/hello
INFO: Elapsed time: 0.532s, Critical Path: 0.29s
INFO: 6 processes: 4 internal, 2 darwin-sandbox.
INFO: Build completed successfully, 6 total actions
iBazel [6:27PM]: Starting...
iBazel [6:27PM]: Error finding workspace: ibazel was not invoked from within a workspace

Kips-MBP:ibazel-no-workspace-file klandergren$ 2024/03/06 18:27:57 hello world
```

**Expected behavior**
`ibazel` to find the workspace root via `MODULE.bazel`

**Actual behavior**
`ibazel` walks the directory tree and does not find the root

**Version:**
 - OS: MacOS 13.6.4
 - Browser: chrome
 - ibazel Version: `iBazel - Version 0.24.0`
 - Bazel version:
```
Bazelisk version: development
Build label: 7.0.2
Build target: @@//src/main/java/com/google/devtools/build/lib/bazel:BazelServer
Build time: Thu Jan 25 16:22:05 2024 (1706199725)
Build timestamp: 1706199725
Build timestamp as int: 1706199725
```
 
**Additional context**
I am new to bazel so if I am mistaken re: `WORKSPACE`/`WORKSPACE.bazel` no longer required please close this issue.

**Comment by jab** [comment]
This issue should be reopened based on interest from the community. Presumably the maintainers would be willing to merge the PR that fixes this (#647) once it's passing CI.

**Comment by alexeagle** [comment]
@achew22 could we re-open this? Bazel 8 was released today, and this issue is the last reason many projects will have a non-functional WORKSPACE file
